frontend/public/components/radio.tsx

```diff
@@ -15,7 +15,12 @@ export const RadioInput: React.SFC<RadioInputProps> = (props) => {
       <label
         className={classNames({ 'radio-inline': props.inline, 'co-disabled': props.disabled })}
       >
-        <input type="radio" {...inputProps} data-test={`${props.title}-radio-input`} />
+        <input
+          type="radio"
+          {...inputProps}
+          data-test={`${props.title}-radio-input`}
+          data-checked-state={`${props.checked}`}
```

Comment:
What you have might be a better approach, but the rest of the `data-checked-state` just dump the raw boolean so like this:
```suggestion
          data-checked-state={props.checked}
```

So I'm thinking consistency is better in this case.  Examples from other checkboxes: [Example 1 (actually a switch)](https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/components/namespace/NamespaceDropdown.tsx#L114) or [Example 2](https://github.com/openshift/console/blob/master/frontend/packages/console-app/src/components/nodes/NodeLogs.tsx#L245)

Reply:
/bugzilla refresh